docs/tutorials/intro.mdx

```diff
@@ -2,11 +2,88 @@
+Velaptor is highly tested.  We maintain a very high level of code coverage to make sure that nothing slips through
+the cracks and are always maintaining and improving our unit tests.
+
+We have achieved this by designing the Velaptor architecture to be loosely coupled as well as making use of the observable
+pattern and dependency injection to make the code as testable as possible.
+
+Manual testing is also performed using a custom testing application that is regularly maintained to catch anything
+that unit tests do not catch.
+
+:::note A quick note about unit testing
+Unit tests are not the end-all-be-all solution for reliable bug-free software.  We do not strive to have
+100% code coverage for the goal of having 100% code coverage.  There are areas of Velaptor that _**CANNOT**_ be tested
+due to some of the interaction with low-level pieces of the framework such as [OpenGL](https://www.opengl.org/), sound,
+and other low-level operating system interactions.  These pieces are abstracted away to achieve high testability
+on the Velaptor side.
+
+This is one reason why we do lots of manual testing to cover the areas that unit tests cannot.
+:::
+
+## Join the community
+
+We are always ready to welcome people to the community.  If you just want to say hi, ask some questions,
+or ask how you can contribute to Velaptor or any of the many KinsonDigital organizational projects, we are here
+to guide you.
+
+- [Discord](https://discord.gg/qewu6fNgv7): For getting help, asking questions, conversation, and where we post updates and releases.
+- [Twitter](https://twitter.com/KDCoder): Where we post updates on releases as well as other software related topics.
+- [KinsonDigital](https://github.com/kinsondigital): The KinsonDigital organization where you can find Velaptor and as well as other projects.
+
+## Learn how to contribute
+
+The Velaptor project is run and maintained by [Calvin Wilkinson](https://twitter.com/KDCoder) and [Kristen Wilkinson](https://twitter.com/kswilky).  We work on this full-time
+and work hard to make this project as well as all of the [KinsonDigital](https://github.com/KinsonDigital) projects in working order.  Any open-source developer will tell you
+that maintaining open-source projects are a ton of work.🥱  
+Any contribution no matter how small is welcome and greatly appreciated.  If you are interested in contributing, refer to
+the links below for more info.
+- Click [here](https://github.com/KinsonDigital/.github/blob/master/docs/CONTRIBUTING.md) to learn how to contribute your time.
+- Click [here](https://github.com/sponsors/KinsonDigital) if you are interested in contributing your funds via one-time or recurring donations.
+
+## License
+
+The Ionic UI Toolkit is a free and open-source project, released under the permissible [MIT license](https://github.com/KinsonDigital/Velaptor/blob/release/v1.0.0/LICENSE.md). This means it can be used in personal or commercial projects for free.
```

Comment:
Grammar
```suggestion
The Ionic UI Toolkit is a free and open-source project, released under the permissible [MIT license](https://github.com/KinsonDigital/Velaptor/blob/release/v1.0.0/LICENSE.md) which means it can be used in personal or commercial projects for free.
```